themes/theme-boilerplate/src/scss/components/_forms.scss

```diff
@@ -265,6 +265,10 @@ div.token-input-dropdown {
     }
 }
 
+.DraftButton {
```

Comment:
Shouldn't this only be applied when the rich editor is on?

Reply:
I checked with @charrondev and it should be hidden all times.